metricflow/plan_conversion/select_column_gen.py

```diff
@@ -32,12 +32,12 @@ def merge(self, other_set: SelectColumnSet) -> SelectColumnSet:
     def as_tuple(self) -> Tuple[SqlSelectColumn, ...]:
         """Return all select columns as a tuple."""
         return tuple(
-            # Preferred order is to have metrics first.
-            self.metric_columns
-            + self.measure_columns
-            + self.dimension_columns
-            + self.time_dimension_columns
+            # Preferred order is to have time_dimensions first.
```

Comment:
If we leave a comment here it should be something more like:

```suggestion
            # This order was chosen to match the column sequence data consumers typically prefer.
```

This tells people of the future that this order was chosen for a reason, which implies any change to the ordering needs justification (either a better reason or else that the order is not correct).